lua/native.lua

```diff
@@ -1,6 +1,6 @@
 ---@meta native
 
---file not used by the game, but used for telling the IDE about the 
```

Comment:
consider `.editorconfig` (https://editorconfig.org/#file-format-details) so your editor will fix it for you